website/app.js

```diff
@@ -24,6 +24,33 @@ function createAposConfig() {
         },
       },
 
+      // Add attachment url hook to replace 'localstack' with 'localhost'
+      '@apostrophecms/attachment': {
+        handlers(self) {
+          return {
+            'apostrophe:modulesRegistered': {
+              fixLocalstackUrl() {
```

Comment:
try to avoid this replacement. You can control this passing `http://localstack:4566/apostrophe-test-bucket` instead of `http://localstack:4566/apostrophe-test-bucket`